internals/setup.js

```diff
@@ -201,6 +223,15 @@ function printFail() {
   const requiredNpmVersion = npm.match(/([0-9.]+)/g)[0]
   await checkNpmVersion(requiredNpmVersion).catch(onError)
 
+  newProjectName = await askUserForNewProjectName()
+  if (newProjectName) {
```

Comment:
What happen if projectName is empty ?

Reply:
If no name is given then no file will be changed.